docs/kubernetes/fsgroup-workaround.md

```diff
@@ -0,0 +1,102 @@
+# CSI driver FsGroup Workaround User Guide
+
+>**Attention:** This workaround is applicable for a cluster 1.19 (CSIVolumeFSGroupPolicy feature gate disabled), and for clusters <= 1.18. When using `stable-master` overlay driver manifest bundle on 1.19 (with CSIVolumeFSGroupPolicy feature gate enabled) and 1.20+ clusters, the workaround is not needed.
```

Comment:
so is there a min version to support this?

Reply:
I have tested the workaround on 1.15 k8s. Havent tested beyond that

Reply:
Cool. I think that's good enough to give us confidence it works almost everywhere. I don't think we should specify a min version though as that implies a certain level of support which we aren't claiming.